src/Arcus.EventGrid/Parsers/EventParser.cs

```diff
@@ -89,5 +93,20 @@ public static EventGridEventBatch<Event> Parse(string rawJson, string sessionId)
             throw new InvalidOperationException(
                 "Couldn't find a correct JSON structure (array or object) to parse the EventGridEvent/CloudEvents from");
         }
+
+        internal static Event ParseJObject(JObject rawInput)
+        {
+            bool isCloudEventV01 = rawInput.ContainsKey(propertyName: "cloudEventsVersion");
+            if (isCloudEventV01)
```

Comment:
What about CloudEvents v1.0? Optionally even v0.2 & v0.3.

Maybe the SDK can help here.

Reply:
Maybe one thing at the time. This PR already adresses two issues.

Reply:
Let's make sure the issue is updated then or have a dedicated one for this.

Reply:
Added: #123